features/storage/TESTS/kvstore/static_tests/main.cpp

```diff
@@ -31,13 +31,20 @@ static char         buffer[20] = {};
 static const size_t data_size = 5;
 static size_t       actual_size = 0;
 static const size_t buffer_size = 20;
+
+/* Adjust number of threads/keys dependent on platform memory resource */
+#if defined(TARGET_NUMAKER_M252KG)
+static const int    num_of_threads = 2;
+static const char   num_of_keys = 2;
+static const char *keys[] = {"key1", "key2"};
+#else
 static const int    num_of_threads = 3;
 static const char   num_of_keys = 3;
+static const char *keys[] = {"key1", "key2", "key3"};
+#endif
 
 static const int heap_alloc_threshold_size = 4096;
```

Comment:
Please try if rising this value would make this device to skip the test.

Reply:
@SeppoTakalo Enlarge `heap_alloc_threshold_size` to 16384 can skip the test. So I submit another PR to replace this one?

Reply:
Modify this PR to use value `4*OS_STACK_SIZE`, which is equivalent.
Then remove other modifications.

Reply:
@SeppoTakalo Do rebase and update as above